app/assets/javascripts/analytics/analytics.js

```diff
@@ -8,10 +23,54 @@ $(function () {
     })
 
     .done(function(data){
-      charts = data.analytics_dashboard_organizer
+      var charts = data.analytics_dashboard_organizer
```

Comment:
Identifier 'analytics_dashboard_organizer' is not in camel case.<br>Missing semicolon.